sdk/communication/azure-communication-callautomation/azure/communication/callautomation/_call_connection_client.py

```diff
@@ -373,7 +380,7 @@ def play_media_to_all(
     @distributed_trace
     def start_recognizing_media(
```

Comment:
playPrompt for recognizing media - does it allow other type of play? such as textSource, like you have added